Source/Core/UICommon/AutoUpdate.cpp

```diff
@@ -20,6 +20,7 @@
 
 #ifdef _WIN32
 #include <Windows.h>
+#include <Common/VCRuntimeUpdater.h>
```

Comment:
Regular quotes, since this is one of our includes (and not a system include)?
```suggestion
#include "Common/VCRuntimeUpdater.h"
```